extension/url-handler.js

```diff
@@ -0,0 +1,98 @@
+
+var urlHandler = {};  // Namespace
+
+/**
+ * In most cases, we don't want to display a full URL with a path.
+ * @param {inputUrl} The original url.
+ * @returns A more minimized version of the url.
+ */
+urlHandler.GetMinimalUrl = function(inputUrl) {
+  var scheme = '';
+  var hostport = '';
+  var parser = document.createElement('a');
+  parser.href = inputUrl;
```

Comment:
The name |parser| sounds slightly misleading, as in you have your own parser. How about just saying |link| or |anchor|?